sites/partners/src/components/listings/PaperListingForm/sections/AdditionalDetails.tsx

```diff
@@ -23,25 +37,46 @@ const AdditionalDetails = (props: AdditionalDetailsProps) => {
         heading={t("listings.sections.additionalDetails")}
         subheading={t("listings.sections.additionalDetailsSubtitle")}
       >
+        <Grid.Row columns={1}>
```

Comment:
question: this isn't mentioned in the ticket so I'll need to get clarity, but is the switch from open text field to multiselect boxes only supposed to be there for non-regulated? If so what should the experience be for regulated listings?